components/inputs/input-date.js

```diff
@@ -260,6 +271,13 @@ class InputDate extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(LitEl
 				</d2l-input-text>
 				${dropdownContent}
 			</d2l-dropdown>
+			${!this._dropdownFirstOpened ? html`<div aria-hidden="true" class="d2l-input-date-hidden-calendar">
```

Comment:
I put this after rather than with the other hidden content because there are some pieces that target `d2l-calendar` and I didn't want to accidentally break anything that would be looking for the first `d2l-calendar`